Queries/Members/Handlers/AutocompleteMember/AutocompleteCounterparty.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+
+namespace Queries.Members.Handlers.AutocompleteMember
+{
+    public class AutocompleteCounterparty
```

Comment:
This could be a record

Reply:
Fixed it ;)

@BerendWouters sorry dat ik mij met u branch moei ma ben ff weggeweest, en dit leek me een taakje wat ik wel kon :)

I hope i did it corectly because i did it with VScode in the browser so i have no omnisharp or .net sdk

Reply:
Nope i did not, see what i mean? 🙃